.github/workflows/frontend-ci.yml

```
@@ -35,10 +38,13 @@ jobs:
     # Permissions are inherited from workflow level
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v4
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)